mapie/conformity_scores/conformity_scores.py

```diff
@@ -369,6 +384,11 @@ def get_bounds(
 
             By default ``False``.
 
+        allow_infinite_bounds: bool
```

Comment:
Why do we call it `allow_infinite_bounds` and the `unbounded`?

Reply:
In `MapieRegressor`, it is helpful to keep `allow_infinite_bounds` to understand what it is. In `get_bounds`, it is not helpful to understand the code because too long.